### PR TITLE
scitools.org.uk modernisation

### PR DESCRIPTION
**Pull request by LukeC92**
This pull request is not actually intended to be merged. This is to show the new version of the site created using mobirise, to allow others to preview it. The mobirise version can be found under
citools.org.uk/SciTools-Dev_2/export
The pages of relevance are index, organisation and privacy. They can be navigated around by opening index.html .

### CODE REVIEW

**Comment by pelson** [comment]
FWIW This can be seen at https://rawgit.com/LukeC92/scitools.org.uk/mobirise_version/SciTools-Dev_2/export/index.html.

**Comment by pelson** [comment]
First impressions - looks great! :+1: 

A few comments, and a few paragraphs that can be slotted in. The paragraphs will be far from perfect, but they are a good start (and we can iterate on them later/after this PR)...

 - [ ] I don't want *existing* URLs to become invalid. Could you map old URLs to the new ones please?

----------

Index - https://rawgit.com/LukeC92/scitools.org.uk/mobirise_version/SciTools-Dev_2/export/index.html:

 - [ ] - menu -> GitHub -> SciTools - if possible, put this link at the bottom of the menu (after a separator) and call it "See All repositories..." (or some such)
 - [ ] - drop the gallery. It is noisy, doesn't add much, and IMO **detracts** from the page. CC @tkknight who may think otherwise
 - [ ] sumpliment typo :rofl: :stuck_out_tongue_winking_eye: 

> SciTools
> Open tools for the analysis and visualisation of Earth science data

> SciTools is a collaborative effort to produce powerful Python-based open-source tools for Earth scientists. Initially started at the Met Office[link] in 2010, SciTools has grown into a diverse community of partners and collaborators from around the world. SciTools is responsible for the maintenance of a number of key packages such as Iris and Cartopy, and continues to develop new and innovate tools for the Earth scientist's toolkit.

 - [ ] Buttons: <Iris> <Cartopy> <cf_units> <Additional Tools>
 - [ ] Make cf_units as prominent as iris and cartopy
 - [ ] put nc_time_axes and scitools/courses in Additional Tools (there will be others too, but don't worry about that yet)

----------

Organisation: https://rawgit.com/LukeC92/scitools.org.uk/mobirise_version/SciTools-Dev_2/export/organisation.html

 - [ ] Organisation doesn't need to be a drop-down. Just one link will do
 - [ ] Drop the paragraph starting "Most often when we say SciTools we are re". The rest is really good.
 - [ ] Run the page through a spell-checker

 - [ ] "One of the primary objectives of this approach is to aid " - not clear what "this approach" refers to
 - [ ] Community forums link will disappear with this change. Need to figure out where that points (the Google Group I assume)



Privacy: https://rawgit.com/LukeC92/scitools.org.uk/mobirise_version/SciTools-Dev_2/export/privacy.html

 - [ ] Address the cookies question

**Comment by LukeC92** [comment]
I'm making most of these changes in mobirise. I'll then copy these over and work on some of the others within a text editor.

Regarding the line "One of the primary objectives of this approach is to aid " In the original SciTools website this line appeared after mentioning that Iris and Cartopy are open source and published under and LGPLv3. For now I have simply swapped around the text order for clarity; we can look into further amendments later.

**Comment by LukeC92** [comment]
The changes I just pushed address some, but not all, of the points @pelson made.

**Comment by pelson** [comment]
Once you're happy with getting these things in @LukeC92, I think the next step is to "de-mobrise" the pages - make sure they have a sensible folder structure, and that only the assets that are needed are included in the repository (currently there is quite a bit of gallery stuff in there still). After that, we can start to refine this in much more detail at the html level.

**Comment by pelson** [comment]
New link: https://rawgit.com/LukeC92/scitools.org.uk/mobirise_version/

**Comment by pelson** [comment]
I took a quick look at the CSS, and think the alignment and spacing of the navbar can be improved with:

```
diff --git a/static/mobirise/css/mbr-additional.css b/static/mobirise/css/mbr-additional.css
index 0c5fec2..5f21666 100644
--- a/static/mobirise/css/mbr-additional.css
+++ b/static/mobirise/css/mbr-additional.css
@@ -1391,10 +1391,10 @@ body {
   position: relative;
   text-decoration: none;
 }
.cid-qQ2bitRL25 .nav-item a {
   display: flex;
   align-items: center;
-  justify-content: center;
+  justify-content: left;
   padding: 0.7rem 0 !important;
   margin: 0rem .65rem !important;
 }
@@ -1468,9 +1468,13 @@ body {
   position: absolute;
   min-width: 5rem;
   padding-top: 1.4rem;
-  padding-bottom: 1.4rem;
+  padding-bottom: 0.5rem;
   text-align: left;
 }
+.dropdown-divider {
+    margin-left: 10%;
+    margin-right: 10%;
+}

```

File: collaboration.html [hunk 1]
```diff
@@ -1,130 +1,9 @@
...
-<p>
-For any potential collaborative development activity it is important that the code base is available to collaborators and licensed to enable collaboration.  Publishing code as it is being developed is an excellent way to work with development communities working in similar spaces.
-</p>
-
-<p>
-It will benefit SciTools development and the development of libraries dependent on SciTools if the code is published and available for developers to investigate and provide feedback on.
-</p>
-
-<p>
-Publishing on a site such as <a href="https://github.com">GitHub</a>, the <a href="governance.html#contributors">SciTools Contributors </a> approach and <a href="http://www.gnu.org/licenses/lgpl.html"> LGPLv3</a> license present one way of managing this. It is the responsibility of a publisher to manage these matters as they see fit, considering license conditions, access rights and the implications for their project.
```
Comment (corinnebosley) [inline collaboration.html]: The 'Scitools Contributors' link doesn't work, takes me to a 404.

**Comment by LukeC92** [comment]
@corinnebosley pointed out a link that no longer works within the sentence 
"Publishing on a site such as GitHub, the SciTools Contributors approach and LGPLv3 license present one way of managing this."
For now I'll just remove the link to "collaboration-governance.html". However the sentence itself doesn't make sense to me. We probably need to read through a lot of the content to ensure clarity.

**Comment by corinnebosley** [comment]
This is great @LukeC92, I think it looks awesome.  It's modern, clear and easily navigable.  Good job.  I'm really impressed.

**Comment by LukeC92** [comment]
I've added some changes to the text, based on the changes @kaedonkers suggested. However there are more changes to commit and I intend to make some comments about the existing text,

File: index.html [hunk 2]
```diff
@@ -1,75 +1,425 @@
...
+              <strong><a href="https://iris-grib.readthedocs.io/en/latest/"
+                 target="_blank"><b>iris-grib</b></a> -</strong> provides
+                 functionality for converting between weather and climate
+                 datasets that are stored as GRIB files and Iris cubes.
+                 <a href="https://iris-grib.readthedocs.io/en/latest/"
+                  target="_blank">Learn more...</a>
+            </li>
+            <li>
+              <a href="https://github.com/SciTools/nc-time-axis"
+                  target="_blank"><strong>nc_time_axes</strong></a> - provides
```
Comment (lbdreyer) [inline index.html]: typo! This should be `nc-time-axis`

File: index.html [hunk 3]
```diff
@@ -1,75 +1,425 @@
...
+                  support for netcdftime axis in matplotlib. <a href=
+                  "https://github.com/SciTools/nc-time-axis"
+                  target="_blank">Learn more...</a>
+            </li>
+            <li>
+              <a href="https://github.com/SciTools/courses"
+                  target="_blank"><strong>scitools/courses</strong></a> -
```
Comment (lbdreyer) [inline index.html]: It doesn't really make sense for this to be under the "Additional Tools" section. Although I do agree that it would be nice to be included on the homepage

File: index.html [hunk 4]
```diff
@@ -1,75 +1,425 @@
...
+                 target="_blank"><b>iris-grib</b></a> -</strong> provides
+                 functionality for converting between weather and climate
+                 datasets that are stored as GRIB files and Iris cubes.
+                 <a href="https://iris-grib.readthedocs.io/en/latest/"
+                  target="_blank">Learn more...</a>
+            </li>
+            <li>
+              <a href="https://github.com/SciTools/nc-time-axis"
+                  target="_blank"><strong>nc_time_axes</strong></a> - provides
+                  support for netcdftime axis in matplotlib. <a href=
```
Comment (lbdreyer) [inline index.html]: Perhaps reword this to be "provides support for non-gregorian datetimes in matplotlib"

File: index.html (hunk 2 above)
Comment (lbdreyer) [inline index.html]: typo! This should be `nc-time-axis`

File: index.html (hunk 3 above)
Comment (lbdreyer) [inline index.html]: It doesn't really make sense for this to be under the "Additional Tools" section. Although I do agree that it would be nice to be included on the homepage

File: organisation.html [hunk 5]
```diff
@@ -0,0 +1,621 @@
...
+                   target="_blank">All Repositories</a>
+            </div>
+          </li>
+        </ul>
+      </div>
+    </nav>
+  </section>
+  <!-- END HEADER -->
+
+  <section class="header1 land-sea-image mbr-parallax-background">
```
Comment (lbdreyer) [inline organisation.html]: This image seems a bit low res. I like the picture, but is there a way to get a higher resolution image?

File: index.html [hunk 6]
```diff
@@ -1,75 +1,425 @@
...
+                <span class="mbri-globe mbr-iconfont mbr-iconfont-btn">
+                </span> Cartopy
+            </a>
+            <a class="btn btn-md btn-primary display-4" href="index.html#additional_tools">
+                <span class="mbri-setting3 mbr-iconfont mbr-iconfont-btn">
+                </span> Additional Tools
+            </a>
+            <a class="btn btn-md btn-primary display-4" href="index.html#cf_units">
+                <span class="mbri-watch mbr-iconfont mbr-iconfont-btn">
+                </span> cf_units
```
Comment (lbdreyer) [inline index.html]: I misunderstood the purpose of these buttons. I see that they're more of a contents rather links other pages as I expected them to be. This does feel a bit unnatural
Regardless, I would remove cf_units as it isn't an individual section

File: index.html [hunk 7]
```diff
@@ -1,75 +1,425 @@
...
+              <h4 class="card-title pb-3 mbr-fonts-style display-7"><a href=
+              "http://scitools.org.uk/iris/docs/latest/index.html"
+                 target="_blank">Iris</a></h4>
+              <p class="mbr-text mbr-fonts-style display-7">The <a href=
+              "http://scitools.org.uk/iris/docs/latest/index.html"
+                 target="_blank">Iris</a> library implements a data model to
+                 create a data abstraction layer which isolates analysis and
+                 visualisation code from data format specifics. The data model
+                 we have chosen is the CF Data Model. The implementation of
+                 this model we have called an Iris Cube.<br>
```
Comment (lbdreyer) [inline index.html]: These sentences are difficult to read and not as accessible as a summary should be, considering people may not familiar with these concepts
I will try to think of a way to reword this

File: index.html (hunk 6 above)
Comment (lbdreyer) [inline index.html]: For each page, these buttons are never on single line. Is it possible to make them be ona single line except (when they don't fit on the screen)

**Comment by pelson** [comment]
Merged in 746974718fb25f9c3bc60680d61d883cb7d51011.

Thanks for this fantastic piece of work @LukeC92.
I'm sure there are a lot of improvements we can continue to make to this, but let's do that iteratively, as we would for normal development (i.e. through small pull requests).

Thanks again @LukeC92!!! :tada:

File: index.html (hunk 7 above)
Comment (pelson) [reply index.html]: Agree completely. I've wanted to do this too. I think whatever we write, we should also put in the iris README.md/rst too.

File: index.html (hunk 3 above)
Comment (pelson) [reply index.html]: Fixed in #198.

File: index.html (hunk 4 above)
Comment (pelson) [reply index.html]: Fixed in #198.

File: index.html (hunk 6 above)
Comment (pelson) [reply index.html]: Fixed in #198.
